lib/buildEmbed.js

```diff
@@ -43,15 +59,29 @@ module.exports = function(tweet, options, index) {
    * Optional embed, which requires a network call to save the Tweet text as static HTML
    * @param {Object} tweet
    * @param {Object} options
-   * @param {Int} index
+   * @param {Number} index
    */
-	async function cachedEmbed(tweet, options) {
+	async function cachedEmbed(tweet, options, index) {
 		const tweetUrl = `https://twitter.com/${tweet.userHandle}/status/${tweet.tweetId}`;
-		const oEmbedUrl = `https://publish.twitter.com/oembed?url=${encodeURIComponent(
-			tweetUrl,
-		)}`;
+		const isScriptEnabled = index === 0 && options.twitterScript.enabled;
+		const oEmbedUrl = new URL('https://publish.twitter.com/oembed');
```

Comment:
This whole method of configuring the oEmbed script is so great! I didn't know `omit_script` even existed 👀 👍🏻